.gitignore

```diff
@@ -12,7 +12,6 @@
 /build
 /dist
 /python/imars3d.egg-info
-conda.recipe/
```

Comment:
You don't have to remove the directory from the `.gitignore`. git will let you add things that are mentioned here if you want. They are default ignored.

Reply:
for some reason, it does not track meta.yaml anymore if I do not remove this line. I am using the web editor for this quick PR.